### PR TITLE
Remove duplicated Quick start images

### PR DESCRIPTION
**Pull request by lvillen**
**What this PR does / why we need it**:

Remove duplicated Quick start images that could potentially be causing an issue with assets handling.

### CODE REVIEW

**Comment by jlledom** [comment]
What's the issue and how removing these images help to solve it?

**Comment by lvillen** [comment]
@jlledom actually this duplicated images is not the cause of the problem, but they're duplicated anyway and we should remove them. Regarding the problem, I'm working on a fix so, if it works, I'll rewrite the PR according to that.

File: package.json [hunk 1]
```diff
@@ -64,10 +64,10 @@
     "@types/lodash.escaperegexp": "^4.1.7",
```
Comment (josemigallas) [inline package.json:64]: Better not to add too many unrelated changes or they will make a mess of git's history.

**Comment by mayorova** [comment]
> What's the problem? How can I reproduce it and check if it's fixed?

The images are not shown in quickstarts (those that have images) in production mode:

image

image

File: package.json
Comment (jlledom) [inline package.json:1]: This is not related to the PR

File: yarn.lock
Comment (jlledom) [inline yarn.lock:1]: No related to the PR

File: config/webpack/environment.js [hunk 2]
```diff
@@ -43,6 +44,13 @@ environment.loaders.append('yaml', {
   type: 'json'
 })
 
+// Handle images coming from the assets folder
+environment.plugins.append('Copy', new CopyWebpackPlugin({
+  patterns: [
+    { from: 'app/assets/images', to: 'images' }
+  ]
+}));
+
```
Comment (jlledom) [inline config/webpack/environment.js:53]: Doesn't this duplicate the images again?

Instead of copying, is there a way to tell webpack to map `images/` to `assets/images/`, and avoid the copy?

**Comment by jlledom** [comment]
Forgot to mention: I tried this branch locally and the error is fixed for me.
